MAUI/DataForm/Working-with-DataForm.md

```diff
@@ -0,0 +1,781 @@
+<td>
+`DataFormAutoCompleteItem`
+</td>
+<td>
+{{'[SfAutoComplete](https://help.syncfusion.com/maui/autocomplete)'| markdownify }}
+</td>
+<td>
+Generated for Enum type property. 
+</td>
+</tr>
+<tr>
+<td>
+DataFormRadioGroupItem
+</td>
+<td>
+{{'[RadioButton](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0)'| markdownify }}
+</td>
+<td>
+Generated for the Enum type property.
+</td>
+</tr>
+</table>
+
+You can customize the property settings or cancel the generation of `DataFormItem` by handling the `GenerateDataFormItem` event.
+
+## Customize auto generated fields
+
+You can customize or cancel the generated `DataFormItem` by handling the `GenerateDataFormItem` event. This event occurs when the field is auto-generated for public and non-static property of the data object.
+
+{% tabs %}
+{% highlight XAML %}
+
+    <?xml version="1.0" encoding="utf-8" ?>
+    <ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+             x:Class="GettingStarted.MainPage">
+    <ContentPage.Content>
+        <dataForm:SfDataForm x:Name="dataForm" GenerateDataFormItem="OnGenerateDataFormItem"/>
+    </ContentPage.Content>
+    </ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+
+    }
+{% endhighlight %}
+{% endtabs %}
+
+`GenerateDataFormItemEventArgs.DataFormItem`  property returns the newly created `DataFormItem`.
+`GenerateDataFormItemEventArgs.DataFormGroupItem`  property returns the newly created `DataFormGroupItem`.
+
+## Cancel DataFormItem generation of the data field
+
+You can cancel the specific `DataFormItem` adding to the data form by handling the `GenerateDataFormItem` event or by defining display attribute to avoid the particular data field being displayed.
+
+#### Using attributes
+
+You can set `AutoGenerateField` to `false` for canceling the `DataFormItem` generation.
+
+{% tabs %}
+{% highlight C# %}
+
+    [Display(AutoGenerateField = false)]
+    public int ID { get; set; } 
+    
+{% endhighlight %}
+{% endtabs %}
+
+#### Using event
+
+In the following code, the `DataFormItem` generation for the `MiddleName` property is canceled by setting the `Cancel` property to true.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem.FieldName == "MiddleName")
+        {
+            e.Cancel = true;
+
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Changing property settings
+
+You can change the property of `DataFormItem` in the `GenerateDataFormItem` event.
+
```

Comment:
<img width="604" alt="image" src="https://user-images.githubusercontent.com/92789666/208892245-66859dbf-7061-4b1e-8b38-a1eed2ec5c65.png">


in the GenerateDataFormItem event when Autogenate true.

For AutoGenerate false you can use property directly by using the DataFormItem - add refrence link for explict create data editor